sdk/identity/identity/interactive-browser-credential.md

```diff
@@ -56,7 +46,6 @@ Then you need to visit your app registration and update the redirect URI you're
 
 ## Sample code
 
-You can see a sample project that uses `InteractiveBrowserCredential` with both [Authorization Code Flow][AuthCodeFlow] and [Implicit Grant Flow][ImplicitGrantFlow] here: [link to the sample project](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/identity/identity/test/manual).
+You can see a sample project that uses `InteractiveBrowserCredential` here: [link to the sample project](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/identity/identity/test/manual).
```

Comment:
Did we update the test at https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/identity/identity/test/manual to no longer show Implicit Grant Flow?

Reply:
Yes! you can see the changes here: https://github.com/Azure/azure-sdk-for-js/pull/14558/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-ee80cf73d7156fa647061c6125cf89f24eb3fd7b4569756f5bcb8343267d3649L9